sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -243,18 +257,10 @@ export interface Entity {
    * Entity sub type, such as Age/Year/TimeRange etc
    */
   subCategory?: string;
```

Comment:
nit: can we change to `subcategory`, it is a complete word.

Reply:
.NET has `SubCategory`. I'll consider doing this rename for GA if we want to align.